suite-common/wallet-types/src/backend.ts

```diff
@@ -36,10 +36,10 @@ export interface Blockchain extends ConnectionStatus {
     explorer: {
         tx: string;
         account: string;
-        queryString: string;
-        nft?: string;
         address: string;
+        nft?: string;
         token?: string;
+        queryString?: string;
```

Comment:
This seems to duplicate the newly created `Explorer` type in `networksConfig`.
I'd suggest exporting `Explorer` and using it here; dependency tree will be unchanged

Reply:
great catch